### PR TITLE
sys-auth/libfprint: add 1.94.5 - sys-auth/pam_fprint: mask for treeclean - sys-auth/fprintd: add 1.94.2, drop really old version

### PR DESCRIPTION
**Pull request by scardracs**
The merge of that PR depends on https://bugs.gentoo.org/888661 (done) and https://bugs.gentoo.org/888673. I will rebase it when these will be done.

Signed-off-by: Marco Scardovi <mscardovi@icloud.com>

### CODE REVIEW

**Comment by Flowdalic** [comment]
Please feel encouraged to squash "[sys-auth/libfprint: drop unused patch](https://github.com/gentoo/gentoo/pull/28865/commits/2d36c4bc61ce617fb362f699f2d91ad28395d235)" into "[sys-auth/libfprint: drop 1.0](https://github.com/gentoo/gentoo/pull/28865/commits/c5b1b564f52439ebb1756ca912f73bcd55ee27e0)".

**Comment by scardracs** [comment]
> Please feel encouraged to squash "[sys-auth/libfprint: drop unused patch](https://github.com/gentoo/gentoo/pull/28865/commits/2d36c4bc61ce617fb362f699f2d91ad28395d235)" into "[sys-auth/libfprint: drop 1.0](https://github.com/gentoo/gentoo/pull/28865/commits/c5b1b564f52439ebb1756ca912f73bcd55ee27e0)".

Done :)

**Comment by scardracs** [comment]
We have never dropped old version libfprint because of pam_fprint, which doesn't work with newer version and they have never tried on updating it. As it has no revdeps I'm gonna mask it

**Comment by scardracs** [comment]
Now it works but actually fprintd's test is broken because of missing pypamtest

**Comment by scardracs** [comment]
Now gdm should be happy again

**Comment by thesamesam** [comment]
(Typo in dependency.)

**Comment by scardracs** [comment]
> (Typo in dependency.)

(Ooops)

File: sys-auth/fprintd/fprintd-1.94.2.ebuild [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~alpha amd64 arm ~arm64 ~ia64 ppc ~ppc64 ~riscv sparc x86"
+IUSE="doc pam systemd test"
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	dev-libs/dbus-glib
+	dev-libs/glib:2
+	>=sys-auth/libfprint-1.94.0:2
```
Comment (thesamesam) [inline sys-auth/fprintd/fprintd-1.94.2.ebuild:25]: Two possible changes:
- I don't know if they always release these in sync (libfprint & fprintd), but if they do, >=sys-auth/libfprint-${PV}:2
- If they don't (I would do this if you're not sure, as it's possible they will at some point), >=sys-auth/libfprint-$(ver_cut 1-2):2

This acts as a nice guard to ensure someone doesn't bump one without the other and the dep stays fresh.

Ignore this if sometimes the versions are totally out of sync/not released at same time.

Comment (scardracs) [reply sys-auth/fprintd/fprintd-1.94.2.ebuild:25]: They used to be synced but I've noticed they aren't since 1.90.x

Comment (thesamesam) [reply sys-auth/fprintd/fprintd-1.94.2.ebuild:25]: OK, fair enough

**Pull request closed without merging**